stm32/unittest/README_unittest.txt

```diff
@@ -49,6 +52,14 @@ Debug and semihosting
 Building and Running the stm32 Unit Tests
 =========================================
 
+0/ Build codec2 for linux and stm32 
+
+   Run the linux cmake build using /path/to/codec2/build_linux 
+   as build directory, see instructions in the codec2 top level directory.
+
+   Run the stm32 cmake build using /path/to/codec2/stm32/build_stm32 
+   as build directory, see instructions in the stm32 directory.
+   
```

Comment:
@donmr @db4ple can you pls confirm that the instructions in codec2/stm32/README build Don's stm32 unit tests now?  Previously these were compiled from codec2/stm32/unittest/src?